documentation/RFC/RFC0005-ETags.md

```diff
@@ -0,0 +1,57 @@
+---
+RFC: RFC0005
+Author: Mark Cowlishaw
+Status: Implemented
+SupercededBy:
+Version: 1.0
+---
+
+# ETag Support
+
+Provide support for ETags in Creation and modification cmdlets, allowing the user to ensure consistency of the resource state under simultaneous update at cloud scale, including the followign functionality.
+
+- Allow the user to get the ETag value of a resource
+- Allow the user to make modifications conditional on the ETag of the resource matching the Etag of the modification request
+- Allow the user to specify destructive changes that ignore ETag settings and overwrite the resource in any state
+
+## Motivation
+
+Explain the benefits of the change for the users.
```

Comment:
Either add motivation, which seems covered above or delete the section / template